cypress.config.js

```diff
@@ -16,7 +16,8 @@ module.exports = defineConfig({
           return {
             username: faker.name.firstName() + `${randomNumber}`,
             email: 'test' + `${randomNumber}` + '@mail.com',
```

Comment:
Using a constant email domain might result in problems with emails being blocked by spam filters. Consider using faker for generating email domains as well.